sklearn/neighbors/_nearest_centroid.py

```diff
@@ -79,6 +84,16 @@ class NearestCentroid(ClassifierMixin, BaseEstimator):
 
         .. versionadded:: 1.0
 
+    priors_ : array-like of shape (n_classes,)
```

Comment:
In naive Bayes models, we name those `class_prior_`. This mighe be better to be aligned with that.